basilisk/tests/test_accessible_output.py

```
@@ -0,0 +1,539 @@
+@pytest.fixture
+def mock_config(mock_config):
+    """Use the shared mock_config fixture and ensure accessible output is enabled."""
+    mock_config.conversation.use_accessible_output = True
+    return mock_config
+
+@pytest.fixture
+def handler():
+    """Create an AccessibleOutputHandler instance for testing."""
+    with patch('basilisk.accessible_output.config.conf') as mock_conf:
+        mock_conf.return_value.conversation.use_accessible_output = True
+        with patch('accessible_output3.outputs.auto.Auto') as mock_auto:
```

Ruff (0.11.9)

Local variable `mock_auto` is assigned to but never used

Remove assignment to unused variable `mock_auto`

(F841)